src/index.js

```diff
@@ -112,22 +109,8 @@ tvArray.forEach(logTV);
 // RMJ8NTvaiiWTSWP5xo3h
 // }
 
-// logLikes();
-
-// document.getElementById('buttonComments1').addEventListener('click', () => {
-//   console.log('button working');
-// });
-
-// console.log(newLike(1));
-// console.log(newLike(2));
-// console.log(newLike(3));
-// console.log(newLike(4));
-// console.log(newLike(5));
-// console.log(newLike(6));
-// newLike(1);
-// newLike(2);
-// newLike(3);
-// newLike(4);
-// newLike(5);
-// newLike(6);
-// console.log(newLike());
+// for (let i = 0; i < responseArray.length; i += 1) {
+//   if (responseArray[i].item_id === `tv${index + 1}`) {
+//
+//   }
+// }
```

Comment:
I would suggest you delete the useless comment for a better reading of the code.